ios/browser/providers/BUILD.gn

```diff
@@ -23,6 +23,7 @@ group("brave_providers") {
     "//ios/chrome/browser/providers/follow:chromium_follow",
     "//ios/chrome/browser/providers/font:chromium_font",
     "//ios/chrome/browser/providers/fullscreen:chromium_fullscreen",
+    "//ios/chrome/browser/providers/google_one:chromium_google_one",
```

Comment:
are we sure whatever this is doing is disabled? @kylehickinson maybe you can follow-up and see if we can resolve this with a stub or something?

Reply:
@bridiver All of these providers by default are already stubs: https://source.chromium.org/chromium/chromium/src/+/main:ios/chrome/browser/providers/google_one/chromium_google_one.mm;l=18

The real implementations are typically in the private `//ios_private` folder specific to Google which they override with the `ios_provider_target` GN arg: https://source.chromium.org/chromium/chromium/src/+/main:ios/public/provider/chrome/browser/build_config.gni;l=23